.gitignore

```diff
@@ -57,3 +57,6 @@ docs/_build/
 
 # OSX Files
 *.DS_Store
+
+# ANAF Data
+l10n_ro_vat_on_payment/data/
```

Comment:
This breaks if the module repo is not writeable by the Odoo user. How about using something like {tools.config['data_dir']}/l10n_ro_vat_on_payment/?